tests/integ/test_stored_procedure.py

```diff
@@ -63,6 +63,14 @@ def setup(session, resources_path):
     )
 
 
+@pytest.fixture(autouse=True)
+def clean_up(session):
+    session.clear_packages()
+    session.clear_imports()
+    session.add_packages("snowflake-snowpark-python")
+    yield
```

Comment:
Do you think it is a better idea to break this down into setup and cleanup? adding inside cleanup looks odd